### DISCUSSION TITLE
WebSite Error

### DISCUSSION THREAD

**Discussion by MeidoNoHitsuji** [GameServerManagers/LinuxGSM#3851] [General]
Why I have Access denied on website https://linuxgsm.com/?

**Comment by Claiyc** [comment]
Access from Russia or Belarus is being denied due to the current political situation. See https://twitter.com/dangibbsuk/status/1498656546850496515?s=21&t=1L8O4vMRjG32Z1YuV9OiQQ

**Reply by popcmp** [reply]
How it help? omg

**Reply by Claiyc** [reply]
You will have to talk to @dgibbs64 about that as it is his project, so his decision. There has already been a very long and extensive discussion about it on our discord server.

**Reply by Claiyc** [reply]
Maybe this statement is helpful; https://github.com/GameServerManagers/LinuxGSM/issues/3853#issuecomment-1121596950

**Comment by Shark-vil** [comment]
> Why I have Access denied on website https://linuxgsm.com/?

Держи, для тех у кого проблемы с доступом:
https://lgsm.vincy.ru/linuxgsm.com/index.html
https://lgsm.vincy.ru/docs/docs.linuxgsm.com/index.html

```
- Sorry, you have been blocked
- You are unable to access linuxgsm.com
- Error 403
- LinuxGSM 403
- Russia
- Russsia LinuxGSM Access
```
